souto-app/src/components/pages/ManageUsers.js

```diff
@@ -0,0 +1,74 @@
+import axios from "axios"
+import React, { Component } from "react"
+import Menu from "../menu"
+import config from "../../config"
+import { MobileStepper, Grid } from "@material-ui/core"
+
+const styles = {
+  progressBar: {
+    width: "100vw",
+    flexGrow: "1",
+    backgroundColor: "transparent",
+    paddingTop: "3vh",
+    width: "90%",
+  },
+  Names: {
+    textAlign: "center",
+  },
+  userProgressInfo: {
+    marginTop: "5vh",
+  },
+  proDiv: {},
+}
+
+export default class ManageUsers extends Component {
```

Comment:
Its a page. So add Pages to the componenet name and the file name. ManageUsersPage.js